WordPress/Classes/Extensions/Colors and Styles/WPStyleGuide+Search.swift

```diff
@@ -24,25 +24,25 @@ extension WPStyleGuide {
 
         // We have to manually tint these images, as we want them
         // a different color from the search bar's cursor (which uses `tintColor`)
-        let cancelImage = UIImage(named: "icon-clear-searchfield")?.imageWithTintColor(.neutral(.shade30))
-        let searchImage = UIImage(named: "icon-post-list-search")?.imageWithTintColor(.neutral(.shade30))
+        let cancelImage = UIImage(named: "icon-clear-searchfield")?.withTintColor(.secondaryLabel)
```

Comment:
I wonder if we should introduce a new named color reference in UIColor+MurielColors called something like `searchBarItem`?

Reply:
I've addressed this and created some new colors for search field customization.